modules/_labs/slider/react/lib/Slider.tsx

```diff
@@ -0,0 +1,264 @@
+import * as React from 'react';
+import {
+  borderRadius,
+  colors,
+  spacingNumbers as spacing,
+  type,
+} from '@workday/canvas-kit-react-core';
+import {focusRing} from '@workday/canvas-kit-react-common';
+
+import styled from '@emotion/styled';
+
+export interface SliderProps {
```

Comment:
Most components merge with a container. This allows applications to send extra attributes to something. I don't know what that should be in this case? The `input[type=range]`? The container?